docs/sources/alerting/monitor/meta-monitoring/_index.md

```diff
@@ -2,7 +2,8 @@
 aliases:
   - meta-monitoring/
   - alerting/meta-monitoring/
```

Comment:
This currently redirects https://grafana.com/docs/grafana/latest/alerting/set-up/alerting/meta-monitoring/ to https://grafana.com/docs/grafana/latest/alerting/set-up/meta-monitoring/.

Keeping the same alias will mean that  https://grafana.com/docs/grafana/latest/alerting/set-up/alerting/meta-monitoring/ will now redirect to your new page location https://grafana.com/docs/grafana/latest/alerting/monitor/meta-monitoring/.

If the original redirect was correct and desirable, then I believe this alias is still correct but I wanted to double check that the original redirect is correct.

Reply:
this is what i want to happen: that this page https://grafana.com/docs/grafana/latest/alerting/set-up/alerting/meta-monitoring/ redirects to your new page location https://grafana.com/docs/grafana/latest/alerting/monitor/meta-monitoring/.

-> what do i need to change?

Reply:
Cool, I believe that this alias is still correct and will redirect https://grafana.com/docs/grafana/latest/alerting/set-up/alerting/meta-monitoring/ to https://grafana.com/docs/grafana/latest/alerting/monitor/meta-monitoring/ once this PR is merged and backported.

Reply:
You can check locally by running `make docs` and browsing to http://localhost:3002/docs/grafana/latest/alerting/set-up/alerting/meta-monitoring/. You should be redirected to your new page. If it 404s, the alias is incorrect.

Reply:
cool, thanks